web/client/actions/backgroundselector.js

```diff
@@ -125,3 +126,10 @@ export function confirmDeleteBackgroundModal(show, layerTitle = null, layerId =
         layerId
     };
 }
+
+export function addBackupBackground(background) {
+    return {
+        type: ADD_BACKUP_BACKGROUND,
+        background
+    };
+}
```

Comment:
This action stashes the selected catalog, not a backup of the  background. 
I'd also document to understand what is the parameter accepted (e.g. ID, object....).
So for this action I should use the following:
```suggestion
/**
 * stashes the current selected catalog to restore after catalog close, when opening
 * the catalog for background selection
 * @params ... <--- Insert here the parameter type and name
 */
export function stashSelectedCatalogService(service) {
    return {
        type: STASH_SELECTED_SERVICE,
        service
    };
}
```